data/json/vehicleparts/stowed_parts.json

```diff
@@ -472,5 +472,73 @@
     "breaks_into": [ { "item": "hd_compressor_unit" }, { "item": "rope_30" } ],
     "flags": [ "EXTRA_DRAG" ],
     "variants": [ { "symbols": "&", "symbols_broken": "*" } ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "stowed_broken_yrax_quadraphract_leg",
+    "name": { "str": "tied-down broken quadraphract leg" },
+    "categories": [ "other" ],
+    "color": "light_blue",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A broken quadraphract leg, secured to the roof for transportation.  Not functional, and hopefully never be.",
+    "item": "broken_yrax_quadraphract_leg",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "35 m", "using": [ [ "chain_material", 3 ] ] }, "removal": { "time": "15 m" } },
+    "breaks_into": [ { "item": "broken_yrax_quadraphract_leg" }, { "item": "chain", "count": [ 3, 3 ] } ],
+    "flags": [ "EXTRA_DRAG" ],
+    "variants": [ { "symbols": "Y", "symbols_broken": "*" } ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "stowed_broken_yrax_quadraphract",
+    "name": { "str": "tied-down broken quadraphract" },
+    "categories": [ "other" ],
+    "color": "light_blue",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "A broken quadraphract, secured to the roof for transportation.  Not functional, and hopefully never be.",
+    "item": "broken_yrax_quadraphract",
+    "location": "on_roof",
+    "requirements": { "install": { "time": "35 m", "using": [ [ "chain_material", 3 ] ] }, "removal": { "time": "15 m" } },
+    "breaks_into": [ { "item": "broken_yrax_quadraphract" }, { "item": "chain", "count": [ 3, 3 ] } ],
+    "flags": [ "EXTRA_DRAG" ],
+    "variants": [ { "symbols": "Y", "symbols_broken": "*" } ]
+  },
+  {
+    "type": "vehicle_part",
+    "id": "stowed_broken_golden_monolith",
+    "name": { "str": "tied-down broken golden monolith" },
+    "categories": [ "other" ],
+    "color": "light_blue",
+    "damage_modifier": 5,
+    "durability": 20,
+    "//": "This is used to simulate a rope being untied/broken and falling off",
+    "description": "More than a ton of gold tied to roof for easier transportation.  Probably the only way to move this without breaking your back.",
```

Comment:
In the vehicle construction menu, does an appropriate lifting tool requirement appear?

Reply:
of course without boom crane you can't lift this thing